plugins/jobtype/src/azkaban/jobtype/tuning/TuningCommonConstants.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2012 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.jobtype.tuning;
+
+/**
+ * Tuning related constants.
+ */
+public class TuningCommonConstants {
```

Comment:
Most of these constants are not related to tuning, e.g. `AZKABAN_EXEC_ID`. It's great that you're pulling these out as constants, but
* Non-tuning related ones should not be in a tuning-specific class
* If these constants are defined elsewhere (including within Azkaban code rather than azkaban-pugin code), you should either re-use that definition or move the definition here. The same constant should not be defined in two different places.

Reply:
Used Azkaban constants wherever required and removed it from TuningCommonConstants.

Reply:
Nice, this looks much better.